### PR TITLE
Feature/adding front

### PR DESCRIPTION
**Pull request by adri-travelperk**
Front end for Recipe app.
The app expects the [django server](https://github.com/adri-travelperk/reciperk) to run on localhost:8000. 

See it in action!
reciperk_front

Future Work:
- Improve unit tests to handle callbacks and react routing.
- Code structure and Clean Code Guidelines.
- Improve error handling.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import RecipeApp from "./RecipeApp";
+
+export default function App() {
```
Comment (ghost) [inline src/App.tsx:4]: Why the choice of returning just a single component, and not add RecipeApp content directly here?

File: src/components/RecipeEditView.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { Grid, Paper, Button } from '@mui/material'
```
Comment (ghost) [inline src/components/RecipeEditView.tsx:2]: We use styled components and a component library called Suitcase, but I think it is fine to use a library for this as long as you know how styled_components are used.

File: src/pages/Dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import Home from './Home'
+import RecipeDelete from './RecipeDelete'
+
+function Copyright(props: any) {
+  return (
+    <Typography variant="body2" color="text.secondary" align="center" {...props}>
+      {'Copyright © '}
+      <Link component={RouterLink} color="inherit" to="https://travelperk.com/">
+        Travelperk
+      </Link>{' '}
```
Comment (ghost) [inline src/pages/Dashboard.tsx:22]: Are these from auto-formatting?

File: src/pages/Dashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
+  );
+}
+
+const drawerWidth: number = 180;
+
+const Drawer = styled(MuiDrawer, { shouldForwardProp: (prop) => prop !== 'open' })(
```
Comment (ghost) [inline src/pages/Dashboard.tsx:31]: Alright I see the styled ones here :)

**Review by ghost — APPROVED** [review]
LGTM!

File: src/App.tsx (hunk 1 above)
Comment (adri-travelperk) [reply src/App.tsx:4]: No strong reason :shrug: 
I just intended to leave the create-react-app stuff as-is and take the actual app to its own folder.

File: src/components/RecipeEditView.tsx (hunk 2 above)
Comment (adri-travelperk) [reply src/components/RecipeEditView.tsx:2]: Awesome!
Yeah, I read that in the onboarding docs after I was done with this PR :facepalm:

File: src/pages/Dashboard.tsx (hunk 3 above)
Comment (adri-travelperk) [reply src/pages/Dashboard.tsx:22]: Yeah, I think so.

**Pull request merged**